### ISSUE TITLE
Add paragraph spacing

### ISSUE THREAD

**Issue by danfitz36** [six7/figma-batch-styler#32]
figma now has paragraph spacing as an option, it'd be nice to be able to update through this plugin

**Comment by danfitz36** [comment]
![Screen Shot 2022-09-13 at 4 28 14 PM](https://user-images.githubusercontent.com/4750495/190002866-0f3faad9-3040-4025-86d4-2dc9188be801.png)
also maybe these options here

**Comment by justinalm** [comment]
Yeah, I’ve just run into this issue making updates to our design system. Having the Paragraph Spacing field in this plugin would be a huge time save.
